### DISCUSSION TITLE
Every third child element with offset (It starts counting from the second one)

### DISCUSSION THREAD

**Discussion by f3bw** [tailwindlabs/tailwindcss#12158] [Help] [answered]
Hello everyone! Is there a way to select every 3rd element in my grid starting from the second element?

The following code has a grid with 3 cols. I want every col in the centre to start lower than the others in the same row. And I want that to repeat every row. So that means every third child. But is starts from the second child.
```
 <div className='grid grid-cols-1 md:grid-cols-3 md:gap-4'>
        {projects.map((project, index) => (
          <ProjectTrigger
            key={index}
            src={project.src}
            url={project.url}
            title={project.title}
            description={project.description}
          />
        ))}
      </div>
```

Does anybody have a nice way of creating this?

**Comment by wongjn** [comment, marked as answer]
You could apply `margin-top` via a `mt-*` class to apply the offset. To match every middle element (2nd, 5th, 8th, etc), you could use some JavaScript logic with `index` to apply the `mt-*` conditionally, or use `:nth-child`: https://play.tailwindcss.com/izOLAoYl1p

**Reply by f3bw** [reply]
As always, thank you @wongjn !
